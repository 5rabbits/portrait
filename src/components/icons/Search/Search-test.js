import { React, toJSON, mount } from 'test/helper'
import Search from 'icons/Search'

describe('Search', () => {
  it('should render with minimum props', () => {
    const component = mount(
      <Search />,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
