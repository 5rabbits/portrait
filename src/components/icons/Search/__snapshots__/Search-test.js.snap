// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search should render with minimum props 1`] = `
<Search>
  <svg
    fill="#000"
    height="1em"
    viewBox="0 0 53.6 53.6"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M53.6 49.4L37.8 33.6C40.4 30 42 25.7 42 21 42 9.4 32.6 0 21 0S0 9.4 0 21s9.4 21 21 21c4.7 0 9-1.6 12.6-4.2l15.8 15.8L53.6 49.4zM2 21C2 10.5 10.5 2 21 2s19 8.5 19 19 -8.5 19-19 19S2 31.5 2 21z"
    />
  </svg>
</Search>
`;
